pkg/server/chaosd/redis.go

```diff
@@ -52,6 +52,16 @@ func (redisAttack) Attack(options core.AttackConfig, env Environment) error {
 
 	case core.RedisSentinelStopAction:
 		return env.Chaos.shutdownSentinelServer(attack, cli)
+
+	case core.RedisCachePenetrationAction:
+		pipe := cli.Pipeline()
+		for i := 0; i < attack.RequestNum; i++ {
+			pipe.Get(cli.Context(), "CHAOS_MESH_nqE3BWm7khHv")
+		}
+		_, err := pipe.Exec(cli.Context())
+		if err.Error() != "redis: nil" {
```

Comment:
how about using `redis.Nil`?
https://redis.uptrace.dev/guide/go-redis.html#redis-nil

Reply:
Updated to `redis.Nil`